src/test/java/seedu/address/model/person/BirthdayTest.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class BirthdayTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Birthday(null));
+    }
+
+    @Test
+    public void constructor_invalidBirthday_throwsIllegalArgumentException() {
+        String invalidBirthday = "";
+        assertThrows(IllegalArgumentException.class, () -> new Birthday(invalidBirthday));
+    }
+
+    @Test
+    public void isValidBirthday() {
```

Comment:
I think this is not a good name for the test case, since you are testing the method `isValidBirthday()`, perhaps something like `testValidBirthday()`

Reply:
Hmm i actually referenced the address test given to us so if we change this I think most of the model attributes test we should change it too. Thoughts?

Reply:
I think @Jiwei99 raised a valid point, we should change the names.